app/report/views.py

```diff
@@ -45,13 +45,13 @@ def get():
                           "active_users": [('', ''), ('o8pj0k', 'John Doe')]}), 200
     """
     agency_ein = request.args.get('agency_ein')
-    user_guid = request.args.get('user_guid')
+    user_guid = request.args.get('user_guid', '')
```

Comment:
Using `None` won't work when user filter is visible and no value is selected because it'll always return empty string as long as `user_guid` exists in `request.args`, even if you set it to `null` in javascript.